addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1990,6 +1990,23 @@ const SnippetOptionWidget = Widget.extend({
             this.$target[0].classList.add(...widgetValue.trim().split(/\s+/g));
         }
     },
+    /**
+     * Default option method which allows to show or hide
+     * the associated snippet in mobile display mode.
+     *
+     * @param {boolean} previewMode - @see this.selectClass
+     * @param {"show"|"hide"} widgetValue
+     * @returns {Promise|undefined}
+     */
+    displayOnMobile: function (previewMode, widgetValue) {
```

Comment:
Doing this you add a whole new method for ALL options. You have to do exactly as you did in the master branch, create a dedicated option for this dedicated method: https://github.com/odoo/odoo/pull/65013/commits/7944e823679871ed8a8d1bd3df00e551d4e77944#diff-70f7fe38208aa7fe678f18e329d3c11b70065dee723921352b6005774e8bab53R1968